ccpp/data/GFS_typedefs.F90

```diff
@@ -753,6 +754,8 @@ module GFS_typedefs
     real(kind_phys)      :: lfnc_k                  !<          Logistic function transition depth (Pa)
     real(kind_phys)      :: lfnc_p0                 !<          Logistic function transition level (Pa)
     logical              :: doGP_lwscat             !< If true, include scattering in longwave cloud-optics, only compatible w/ GP cloud-optics
+    logical              :: doGP_sgs_cnv            !< If true, include SubGridScale convective cloud in RRTMGP
+    logical              :: doGP_sgs_mynn           !< If true, include SubGridScale MYNN-EDMF cloud in RRTMGP 
```

Comment:
For the SG MYNN cloud related variables, for example, doGP_sgs_mynn and cld_mynn_lwp, I suggest to change them to doGP_sgs_pbl and cld_pbl_lwp, because those variables could be used for other PBL schemes as well.

Reply:
@ChunxiZhang-NOAA 
I understand changing doGP_sgs_mynn -> doGP_sgs_pbl, but why doGP_sgs_cnv -> cld_pbl_lwp?

Reply:
@dustinswales  I mean doGP_sgs_mynn -> doGP_sgs_pbl;  cld_mynn_lwp -> cld_pbl_lwp. Sorry for the confusing.

Reply:
Excellent idea.
Done